### ISSUE TITLE
Edited messages are not updated in PNs at all and in pinned messages until chat view reload

### ISSUE THREAD

**Issue by churik** [status-im/status-react#15651]
# Bug Report

## Problem 
When editing a message, expected that it will be updated on the receiver side everywhere. Currently push notification remains the same and pinned messages list is updated only after chat-reopening

#### Expected behavior
the receiver can see updated messages

#### Actual behavior
1) PN is not updated: 

https://user-images.githubusercontent.com/4557972/232010780-9d6076a9-e2ec-4df5-a30c-8bb29da09e63.mp4 

(`bbbb` instead `bbbb edit`)

### Reproduction
**Prerequisites:** user A and User B are contacts, User B has PNs enabled 

1) User A: send message to user B
2) User A: pin message
3) User A: edit message
4) User B: check message in pin messages list and check PN

### Additional Information

- Status version: nightly 14/03/23
- Operating System: Android, iOS

[comment]: # (Please, add logs/notes if necessary)

**Comment by ilmotta** [comment]
@ulisesmac I moved this issue to the backlog and out of the 2.32 milestone because push notification bugs & features should be momentarily descoped. We will come back to PNs in 2.33+.
